### ISSUE TITLE
Missing Roles

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#494]
Issue group: Header
Visibility: 3
Evaluation: Partially Supports
Success Criteria: 4.1.2: Name, Role, Value
Needs design: 

### Description
The More submenu in the navigation on medium sized screens uses a <div> without providing any roles. aria-expanded can only be applied to interactive elements with roles such as button or link. This can be addressed by using a <button> instead of a <div> similar to the Your account menu or by applying role=“button” to  the element. Another option would be to remove this submenu completely, as it appears that the only item, Settings, would fit in the site navigation at the medium screen width.

![](https://dl.airtable.com/.attachments/2bcdce64db06523ce5bbbc217ab975fc/bff0702b/Capturedecranle2022-03-15a13_58_29.png)

**Comment by mtoutloff** [comment]
Was in new issues and pulled in to Review and Phil was going to take a look to see if it's fixed

**Comment by amazingphilippe** [comment]
We apply the aria-expanded role to a link, which is an interactive component and not a div. This is fixed and can be closed.
